onboard/src/data_pub/data_pub/serial_republisher_node.py

```diff
@@ -52,13 +52,15 @@ def connect(self):
         Read FTDI strings of all ports in list_ports.grep
         """
         try:
-            self._serial_port = next(list_ports.grep(self._config_data[self._config_name]['ftdi'])).device
-            self._serial = serial.Serial(self._serial_port, self.baud,
+            self._serial_port = next(list_ports.grep(self._config_data[self._config_name]['ftdi'])).device.strip()
+            print(f"{self._config_name} sensor found at {self._serial_port} Connecting with baud {self._baud}.")
+            self._serial = serial.Serial(self._serial_port, self._baud,
                                             timeout=1, write_timeout=None,
                                             bytesize=serial.EIGHTBITS, parity=serial.PARITY_NONE,
                                             stopbits=serial.STOPBITS_ONE)
-            self.run_timer.reset()
             self.connect_timer.cancel()
+            self.run_timer.reset()
+            print(f"Connected to {self._config_name} sensor at {self._serial_port}.")
```

Comment:
Python print or ros log?

Reply:
should be good just to only print for debugging this wont print when launch filed